MSBuild.Sdk.SqlProj.sln

```diff
@@ -10,6 +10,8 @@ EndProject
 Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Solution Items", "Solution Items", "{D25C8812-D92E-41D5-9BC9-30AFCF45063B}"
 	ProjectSection(SolutionItems) = preProject
 		.editorconfig = .editorconfig
+		.github\workflows\main.yml = .github\workflows\main.yml
+		README.md = README.md
```

Comment:
Is adding these files to the solution file part of this change? I don't think I understand what adding them does.

Reply:
It just makes the files visible in solution explorer in VS so they are less likely to be overlooked/forgotten